tests/unit/reports/single_table/test_single_table_quality_report.py

```diff
@@ -188,10 +188,10 @@ def test_generate(self):
         # Assert
         mock_validate.assert_called_once_with(real_data, synthetic_data, metadata)
         quality_report._properties['Column Shapes'].get_score.assert_called_with(
-            real_data, synthetic_data, metadata
+            real_data, synthetic_data, metadata, progress_bar=None
```

Comment:
Let's add a unit test with the progress bar actually being created (we can mock it) to just check it's being called with the expected values

Reply:
Yes done in [a817ed6](https://github.com/sdv-dev/SDMetrics/pull/378/commits/a817ed6a5f9334832dab7af44592333cb0ff2562)